packages/eslint-config-pwa-kit/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "eslint-config-pwa-kit",
+  "version": "2.0.0-dev.6",
+  "engines": {
+    "node": "^12.0.0 || ^14.0.0",
+    "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+  },
+  "description": "ESLint config used by PWA Kit",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+  },
+  "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/eslint-config-pwa-kit#readme",
+  "bugs": {
+    "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "author": "dev@mobify.com",
+  "main": "index.js",
+  "scripts": {
+    "lint": "npm run lint:js",
+    "lint:fix": "npm run lint:js -- --fix",
+    "lint:js": "cross-env ../../node_modules/.bin/eslint \"**/*.{js,jsx}\""
+  },
+  "dependencies": {
+    "@babel/core": "^7.17.9",
+    "@babel/eslint-parser": "^7.17.0",
+    "@rushstack/eslint-patch": "^1.1.3",
+    "@typescript-eslint/parser": "^5.20.0",
+    "babel-preset-pwa-kit": "^2.0.0-dev.6",
```

Comment:
Do we need it as dependency here? 🤔

Reply:
Reason I asked is if `babel-preset-pwa-kit` is a dependency of `eslint-config`. When a generated project install the `eslint-config`, the babel preset packages should also be installed as well. 

With that logic, do we still need to have `babel-preset-pwa-kit` to be listed as a dependency in each template-*?

Reply:
To verify the eslint still works without explicitly having `babel-preset-pwa-kit`. what I did:
-  Removed the `babel-preset-pwa-kit` [here](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/529/files#diff-00e24db644be6a5f62f06ee3d5eb644005bdb9590ebc8ddf6a3c03d1b82fa6ebR25) from `template-retail-react-app`. 
- Generated the project with command 
` GENERATOR_PRESET=test-project node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir ../generated-project-no-babel`
- Opened the generated project with my IDE, the eslint works as expected without throwing any issue
![image](https://user-images.githubusercontent.com/52219283/166352946-bb211c8e-4ac2-491e-96f2-d12d95a17758.png).

From this, either we
- Have `babel-preset-pwa-kit` as dependency, and do not need to install it inside `template-*`
- Do not include it as dependency here, only peerDependency, and have it as dependencies in template-*